src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -319,96 +319,109 @@ protected void registerRecipes(Consumer<IFinishedRecipe> recipeSaver)
 		GristCostRecipeBuilder.of(MSItems.GOLDEN_SPORK).grist(BUILD, 70).grist(GOLD, 40).grist(DIAMOND, 1).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.BIDENT).grist(BUILD, 239).grist(GARNET, 50).grist(AMETHYST, 50).grist(QUARTZ, 30).build(recipeSaver);
-		
-		GristCostRecipeBuilder.of(MSItems.BATLEACKS).grist(BUILD, 0).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.BIDENT).grist(BUILD, 12390).grist(GARNET, 2000).grist(AMETHYST, 2050).grist(QUARTZ, 1300).build(recipeSaver);
+
+		GristCostRecipeBuilder.of(MSItems.BATLEACKS).grist(ARTIFACT, -10).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSItems.COPSE_CRUSHER).grist(BUILD, 25).grist(RUST, 15).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.QUENCH_CRUSHER).grist(BUILD, 125).grist(IODINE, 34).grist(CAULK, 23).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.MELONSBANE).grist(BUILD, 245).grist(IODINE, 67).grist(CAULK, 63).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.QUENCH_CRUSHER).grist(BUILD, 125).grist(IODINE, 74).grist(CAULK, 53).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.MELONSBANE).grist(BUILD, 545).grist(IODINE, 170).grist(CAULK, 130).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSItems.CROP_CHOP).grist(BUILD, 130).grist(IODINE, 93).grist(AMBER, 72).grist(GOLD, 45).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.THE_LAST_STRAW).grist(BUILD, 270).grist(GOLD, 124).grist(IODINE, 103).grist(AMBER, 95).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.BATTLEAXE).grist(BUILD, 400).grist(RUST, 240).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.CANDY_BATTLEAXE).grist(IODINE, 314).grist(GOLD, 240).grist(CHALK, 160).grist(AMBER, 98).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.CHOCO_LOCO_WOODSPLITTER).grist(BUILD, 300).grist(IODINE, 265).grist(GOLD, 200).grist(CHALK, 120).grist(AMBER, 41).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.STEEL_EDGE_CANDYCUTTER).grist(BUILD, 500).grist(RUST, 413).grist(IODINE, 212).grist(GOLD, 167).grist(CHALK, 120).grist(AMBER, 43).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.BLACKSMITH_BANE).grist(BUILD, 30).grist(RUST, 15).grist(TAR, 12).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.REGIAXE).grist(AMETHYST, 200).grist(TAR, 146).grist(GOLD, 98).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.SURPRISE_AXE).grist(BUILD, 170).grist(RUST, 80).grist(AMBER, 30).grist(IODINE, 12).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.SHOCK_AXE).grist(BUILD, 275).grist(RUST, 125).grist(GOLD, 121).grist(AMBER, 90).grist(URANIUM, 50).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.SCRAXE).grist(BUILD, 139).grist(TAR, 86).grist(RUST, 43).grist(RUBY, 8).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.LORENTZ_DISTRANSFORMATIONER).grist(BUILD, 150).grist(URANIUM, 20).grist(TAR, 20).grist(RUST, 15).build(recipeSaver);
```

Comment:
So I think you accidentaly removed this grist cost

Reply:
Yep. I'll fix that.